scripts/system/more/more.html

```diff
@@ -0,0 +1,276 @@
+<!DOCTYPE html>
+<!--
+//  more.html
+//
+//  Created by Keb Helion, February 2020.
+//  Copyright 2020 Project Athena and contributors.
+//
+//  App maintained in: https://github.com/kasenvr/community-apps
+//  App copied to: https://github.com/kasenvr/project-athena
+//
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+-->
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <script id="metadataScriptTag" type="text/javascript" src="https://kasenvr.github.io/community-apps/applications/metadata.js"></script>
+        <script>
+            //Defaults
+            var DEFAULT_PER_PAGE = 3;
+            var DEFAULT_OFFSET = 0;
```

Comment:
These could be `const`.

Reply:
Making it const caused critical system failure (aka it stopped working) so let's avoid that for now.